[New Product] TrueNAS

harryzcy opened the pull request:
Adds a product page for TrueNAS, the Debian-based NAS OS from iXsystems (formerly TrueNAS Scale, renamed "Community Edition" in Jan 2025). Covers 23.10 through 25.10.

No support policy is documented. Per the [software status page](https://www.truenas.com/docs/softwarestatus/), only the two most recent releases are maintained, and the policy is the same for both editions. Only 24.10 has a stated end date, so 24.04 and 23.10 use `eol: true`.

No auto-update: there are no GitHub Releases.

Closes #3048

harryzcy committed:
[New Product] TrueNAS

harryzcy committed:
Merge branch 'master' into truenas

harryzcy committed:
Merge branch 'master' into truenas

harryzcy committed:
Merge branch 'master' into truenas

harryzcy committed:
Merge branch 'master' into truenas

harryzcy committed:
Merge branch 'master' into truenas

marcwrobel commented on products/truenas.md:
```diff
@@ -0,0 +1,46 @@
+    latest: "25.04.2.6"
+    latestReleaseDate: 2025-10-30
+
+  - releaseCycle: "24.10"
```
Can a add one or two EOL releases ?

harryzcy replied:
Added two EOL releases

harryzcy committed:
Update latest release

harryzcy committed:
Add two eol releases

harryzcy commented on products/truenas.md:
```diff
@@ -0,0 +1,46 @@
+  - releaseCycle: "24.10"
+    codename: "Electric Eel"
+    releaseDate: 2024-10-29
+    eol: false
```
Yeah, the website does not clearly document that last three releases are supported. But based on the [the second link you mentioned](https://www.truenas.com/docs/softwarestatus/#release-schedule), only last three are marked as "Maintenance" status. I would say that's the case.

marcwrobel replied:
I think it would be nice to document this somehow, similar to https://endoflife.date/liquibase. WDYT ?

harryzcy replied:
I've added the section that "support policy is not documented..."

harryzcy commented on products/truenas.md:
```diff
@@ -0,0 +1,46 @@
+---
+> [TrueNAS](https://www.truenas.com/) is a family of enterprise network-attached storage (NAS)
+> products developed by iXsystems Inc. based on Debian. TrueNAS is released in two editions: 
+> Enterprise and Community Edition.
```
What I understand is that support policy is the same. Enterprise only enables more features and SLA guarantee. But both are from the same release.

harryzcy committed:
Use release notes as the link

harryzcy committed:
Include initial release data for eol releases

harryzcy committed:
Merge branch 'master' into truenas

harryzcy committed:
Merge branch 'master' into truenas

harryzcy committed:
Merge branch 'master' into truenas

harryzcy committed:
Merge branch 'master' into truenas

marcwrobel commented on products/truenas.md:
```diff
@@ -0,0 +1,62 @@
+> Enterprise and Community Edition.
+
+Historically, iXsystems provides a FreeBSD-based TrueNAS Core but it's discontinued and entered into maintenance mode.
+TrueNAS Scale, a Debian-based OS is released in 2022 and is the only actively maintained offering going forward. In January
```
```suggestion
TrueNAS Scale, a Debian-based OS was released in 2022 and is the only actively maintained offering going forward. In January
```

harryzcy committed:
Update products/truenas.md

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

harryzcy committed:
Say support policy not documented

harryzcy committed:
Some formatting

harryzcy committed:
Merge branch 'master' into truenas

harryzcy committed:
Update TrueNAS 25.10 latest and mark 24.10 EOL

25.10.5 was released on 2026-07-23. The previous entry paired 25.10.3
with 2025-12-18, which is 25.10.1's release date.

The 24.10 documentation is now archived and states the cycle received
its final update on 2025-08-07, with 24.10.2.4 as the terminal version.

harryzcy committed:
Say two most recent TrueNAS releases are maintained

24.10 reached end of life on 2025-08-07, leaving 25.10 and 25.04 as the
only maintained releases.

marcwrobel reviewed (approved):
Thanks @harryzcy for this PR.

I have a few minor suggestions. I will apply them myself so that this can be merged.

marcwrobel committed:
Apply suggestions from code review

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

marcwrobel committed:
Apply suggestion from @marcwrobel
